nft-rfc-008.md

```diff
@@ -55,117 +55,111 @@ content of a verifiable claim, to determine whether the claim meets specific acc
-   to the processed VC2s, along with CarbonAudit’s certifying VC4 that can be used to verify their standing with the UNFCCC, into a VC3 which 
-   is returned to HydroElec Inc. 
-8. Once HydroElec Inc receives a VC3 which asserts CarbonAudit’s verification of RECs worth the equivalent of 100 tons worth of carbon, 
-   the VC4 is passed to the UNFCCC for final evaluation.
-9. The UNFCCC reviews the submitted VC3, including verifying the referenced certifying agencies issued VC1 (from Sustained Certs) and 
-   VC4 (from UNFCCC), and once they are satisfied that all is in order, bundle VC4 with a verification assertion into VC5. 
-10. VC5 is then minted into a Carbon Credits NFT which represents 100 tons worth of Carbon Credits. A capability to retrieve this minted NFT 
-    is then given to HydroElec Inc by the UNFCCC.
-11. HydroElec Inc claims their Carbon Credit NFT, granting them full ownership.
+* __Clean Energy Blockchain__ is a ficticious blockchain where HydroElec will register their issued clean energy claims by reference. The
+  claims themselves are stored in a secure data vault controlled by HydroElec which needs a capability to access. 
+
+1. The UNFCCC publishes a new NFT schema for RECs which includes a set of criteria--as listed below--as well as specifying that producing 
+   a verifiable certificate from an approved site auditor (CleanEnergy Certifier) can be exchanged for the capability to mint REC NFTs:
+   * Claim Issuer must have their facilities certified by a certifying agency recognized by the UNFCCC
+   * Claim Issuer must have their claims validated by a Claim Verification Agent that is recognized by the UNFCCC
+      - Claim Verification Agent must include a hashed reference to each Claim they processed
+      - Claim Verification Agent must include a timestamp in ISO 8601 format for when each Claim was processed
+   * Energy Claims must be made in units of MWh
+   * Energy Claims must include the time range the energy was produced in and must follow ISO 8601
+   * Claim Issuer must include a specific IOT device reference the claim was generated from
+   * Claim Issuer must include last service date in ISO 8601 format for each IoT device
+2. CleanEnergy Certifier gets certified by the UNFCCC to act as a project auditor for the purposes of certifying a project to be able to make
+   REC claims and receives VC4 from the UNFCCC. At the same time CarbonAudit gets certified by the UNFCCC as a Claim Verification Agent for 
+   these new REC NFTs and receives a capability from UNFCCC to issue an Evaluation Report Credentials (VC3). 
+3. HydroElec get CleanEnergy Certifier to audit the new electric dam they just converted and receives a Certifying Credential (VC1) from 
+   CleanEnergy Certifier which includes, by reference, CleanEnergy Certifier's Certification Credential (VC4) they received from UNFCCC.
+4. HydroElec submits their Certifying Credential (VC1) to the UNFCCC and receives the capability of minting REC NFTs. 
```

Comment:
This step might be redundant and can be further decentralised, by adding a `caveat` to the authorisation provided by UNFCC to the producer, that they must present proof of VC1 to be able to `evokeCapability` `mintNft`